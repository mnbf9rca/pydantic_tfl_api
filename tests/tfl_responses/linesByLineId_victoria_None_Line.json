{"status_code": 200, "headers": {"Date": "Mon, 15 Jul 2024 15:02:46 GMT", "Content-Type": "application/json; charset=utf-8", "Content-Length": "265", "Connection": "keep-alive", "Cache-Control": "public, must-revalidate, max-age=43200, s-maxage=86400", "Via": "1.1 varnish", "Content-Encoding": "gzip", "Accept-Ranges": "bytes", "Age": "82708", "Access-Control-Allow-Headers": "Content-Type", "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS", "Access-Control-Allow-Origin": "*", "Api-Entity-Payload": "Line", "X-Backend": "api", "X-Cache": "HIT", "X-Cache-Hits": "12", "X-Cacheable": "Yes. Cacheable", "X-Frame-Options": "deny", "X-Proxy-Connection": "unset", "X-TTL": "86400.000", "X-TTL-RULE": "0", "X-Varnish": "905617630 892920795", "X-AspNet-Version": "4.0.30319", "X-Operation": "Line_GetByPathIds", "X-API": "Line", "CF-Cache-Status": "DYNAMIC", "Strict-Transport-Security": "max-age=31536000", "Set-Cookie": "_cfuvid=o3vjlhNrL_XP3x2UX9OQbmOBZ_4Ft78MpxVqSS.LJSo-1721055766959-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None", "Server": "cloudflare", "CF-RAY": "8a3aa2af19438883-LHR"}, "url": "https://api.tfl.gov.uk/Line/victoria", "content": "[{\"$type\":\"Tfl.Api.Presentation.Entities.Line, Tfl.Api.Presentation.Entities\",\"id\":\"victoria\",\"name\":\"Victoria\",\"modeName\":\"tube\",\"disruptions\":[],\"created\":\"2024-07-09T13:08:01.79Z\",\"modified\":\"2024-07-09T13:08:01.79Z\",\"lineStatuses\":[],\"routeSections\":[],\"serviceTypes\":[{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Regular\"},{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Night\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Night\"}],\"crowding\":{\"$type\":\"Tfl.Api.Presentation.Entities.Crowding, Tfl.Api.Presentation.Entities\"}}]"}