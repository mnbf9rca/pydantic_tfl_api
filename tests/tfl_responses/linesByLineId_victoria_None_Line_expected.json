["{\"id\":\"victoria\",\"name\":\"Victoria\",\"mode_name\":\"tube\",\"disruptions\":[],\"created\":\"2024-07-09T13:08:01.790000Z\",\"modified\":\"2024-07-09T13:08:01.790000Z\",\"line_statuses\":[],\"route_sections\":[],\"service_types\":[{\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Regular\"},{\"name\":\"Night\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Night\"}],\"crowding\":{\"passenger_flows\":null,\"train_loadings\":null},\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}"]